src/main/g8/build.sbt

```diff
@@ -10,7 +10,7 @@ import scala.scalanative.build._
 
 // defaults set with common options shown
 nativeConfig ~= { c =>
-  c.withLTO(LTO.none) // thin
-    .withMode(Mode.debug) // releaseFast
-    .withGC(GC.immix) // commix
+  c.withSourceLevelDebuggingConfig(
```

Comment:
Perhaps the current `nativeConfig` section, with the alternatives
can be preserved and the right hand comment changed to something 
like "// frequent alternative 'foo'".

The problem with minimalism, especially for people just joining the party, is the balance between how may people  a comment gains and how many people it looses/confuses. I'd ask to be given Wisdom but
I am afraid that is something that must be earned.

As one voice among many, I like having that section, with _some_ alternatives and use both it and the alternative quick reference all the time "now, once again, is that releaseFast or release-fast?"